src/Sarif.Viewer.VisualStudio/SarifSnapshot.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Windows.Forms;
+
+using EnvDTE;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Driver.Sdk;
+using Microsoft.CodeAnalysis.Sarif.Sdk;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Shell.TableManager;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace SarifViewer
+{
+    class SarifSnapshot : TableEntriesSnapshotBase
```

Comment:
This class could use a comment IMO. I don't know what a "snapshot" is, and a quick scan of the class didn't reveal to me what it was for.